<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDataDialog_3</class>
 <widget class="QDialog" name="addDataDialog_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加故障记录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>162</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>故障模式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>162</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>故障部位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>162</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>故障原因：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>162</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>故障溯源：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>162</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>整机/子系统：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>整机</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>子系统</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addDataDialog_3</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addDataDialog_3</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
